src/parquet/arrow/reader.cc

```diff
@@ -204,9 +214,21 @@ class FileReader::Impl {
   int num_threads_;
 };
 
+typedef const int16_t* ValueLevelsPtr;
```

Comment:
With that naming I was trying to mimic other typedef and hide implementation details, but when you mention it - maybe visibility is better here? I don't have a solid opinion here.. I'll make the change if you think it makes sense.